# HG changeset patch
# User Jacob Dawid <jacob.dawid@cybercatalyst.net>
# Date 1345284066 -7200
#      Sat Aug 18 12:01:06 2012 +0200
# Node ID 0f0b795044c3ab868bc950f79953a6310e6def87
# Parent  2d5e4d283688243cfe3337de8085ab6f3b43bdfc
Moved translators file and added GUI structure to etc/HACKING.

* translators: Moved to languages subdirectory.
* HACKING: Added GUI structure.

diff --git a/etc/HACKING b/etc/HACKING
--- a/etc/HACKING
+++ b/etc/HACKING
@@ -170,17 +170,29 @@ An overview of the directory layout of O
                      operators for all possible Octave type
                      combinations.
     TEMPLATE-INST  * Some C++ template instantiations.
 
   test          -- tests for the interpreter
     test_*.m       * fixed tests for the interpreter
     fntests.m      * script to run function tests embedded in C++ and .m files
 
-
+  gui           -- the graphical user interface of GNU Octave
+    src            * source files.
+      m-editor     * source files for the m-file editor.
+      octave-adapter * souce files for the octave layer for threadsafe
+                     communication with the octave interpreter.
+      qtinfo       * source files for the Qt texinfo browser.
+      icons        * icon files that will be compiled into the executable via
+                     a resource file.
+    qterminal      * subrepository for the qterminal Qt terminal widget.
+    languages      * translation files and list of translators.
+    kb-layouts     * various files need by the qterminal widget that has been
+                     derived from Konsole.
+      
 ----
 John W. Eaton
 jwe@octave.org
 
 
 Last updated: Tue Nov 22 20:51:34 PST 2011
 
 
diff --git a/gui/translators b/gui/languages/translators
rename from gui/translators
rename to gui/languages/translators
